R/finalsize-package.R

```diff
@@ -0,0 +1,5 @@
+#' @useDynLib finalsize
+#' @import RcppEigen
+#' @importFrom Rcpp evalCpp
+#' @exportPattern "^[[:alpha:]]+"
```

Comment:
Can we remove this as well? I think it's safer to export select objects.

Reply:
Removed in https://github.com/epiverse-trace/finalsize/pull/36/commits/940fb38c0814679a04b9d5f9af2c393378952f23